### PR TITLE
AST-71 - fix: Mega menu heading hover color issue

### PR DESCRIPTION
**Pull request by abhishek-bsf**
### Description

Task- https://wp-astra.atlassian.net/browse/AST-71

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests

### CODE REVIEW

File: inc/class-astra-dynamic-css.php [hunk 1]
```diff
@@ -542,7 +542,7 @@ public static function return_output( $dynamic_css, $dynamic_css_filtered = '' )
 					'color' => esc_attr( $text_color ),
 				);
 				// Main - Menu Items.
-				$css_output['.main-header-menu .menu-item:hover > .menu-link, .main-header-menu .menu-item:hover > .ast-menu-toggle, .main-header-menu .ast-masthead-custom-menu-items a:hover, .main-header-menu .menu-item.focus > .menu-link, .main-header-menu .menu-item.focus > .ast-menu-toggle, .main-header-menu .current-menu-item > .menu-link, .main-header-menu .current-menu-ancestor > .menu-link, .main-header-menu .current-menu-item > .ast-menu-toggle, .main-header-menu .current-menu-ancestor > .ast-menu-toggle'] = array(
+				$css_output['.main-header-menu .menu-item .menu-link:hover, .main-header-menu .menu-item:hover > .ast-menu-toggle, .main-header-menu .ast-masthead-custom-menu-items a:hover, .main-header-menu .menu-item.focus > .menu-link, .main-header-menu .menu-item.focus > .ast-menu-toggle, .main-header-menu .current-menu-item > .menu-link, .main-header-menu .current-menu-ancestor > .menu-link, .main-header-menu .current-menu-item > .ast-menu-toggle, .main-header-menu .current-menu-ancestor > .ast-menu-toggle'] = array(
```
Comment (premanshup) [inline inc/class-astra-dynamic-css.php:545]: @abhishek-bsf If the issue is in Mega Menu, why are we changing the CSS of main menu?

**Review by premanshup — CHANGES_REQUESTED** [review]

File: inc/class-astra-dynamic-css.php (hunk 1 above)
Comment (abhishek-bsf) [reply inc/class-astra-dynamic-css.php:545]: This CSS selector is causing issue while applying CSS to Mega menu. It overrides Mega menu styles and dynamic Mega menu style is not applied. I have tested this change and it does not affect any Main menu style.

**Review by imnavanath — APPROVED** [review]

File: inc/class-astra-dynamic-css.php (hunk 1 above)
Comment (imnavanath) [reply inc/class-astra-dynamic-css.php:567]: It will be better if we test this case.

Comment (abhishek-bsf) [reply inc/class-astra-dynamic-css.php:567]: @imnavanath I tested this case. Basically in this ``.menu-item:hover > .menu-link`` we were adding hover action for ``<li>`` tag which was causing some issue to apply style. Instead now, we have directly targeted ``<a>`` tag class for hover style ``.menu-link:hover``.

**Pull request closed without merging**